apps/andi/assets/css/app.scss

```diff
@@ -48,7 +48,7 @@ body {
 
 .error-msg {
   color: red;
-  display: block;
+  display: block !important;
```

Comment:
Why are you using !important?

Reply:
yeah we kind of just gave up going through the cascade but we will change this back